src/CreateMikLabelModel/DL/DownloadHelper.cs

```diff
@@ -123,7 +121,9 @@ public static async Task<int> DownloadItemsAsync(string outputPath, (string owne
                         {
                             var prsWithTooManyFileChanges =
                                 issuePage.Issues.Repository.Issues.Nodes
-                                    .Where(x => x as PullRequestsNode != null).Select(x => x as PullRequestsNode).Where(i => i.Files != null && i.Files.TotalCount > MaxFileChangesPerPR);
+                                    .Where(x => x as PullRequestsNode != null)
+                                    .Select(x => x as PullRequestsNode)
```

Comment:
I think this .Where and .Select can both be replaced by 1 line:

```c#
     .OfType<PullRequestsNode>()
```

This will return a new strongly-typed enumerable of only the `PullRequestsNode` items.